go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/teamwork/test v0.0.0-20181126061546-2ff8918eb6a4
 	github.com/teamwork/utils v0.0.0-20190114034940-d6a1f27ce92c
-	gopkg.in/yaml.v2 v2.2.2
+	gopkg.in/yaml.v2 v2.4.0 // indirect
```

Comment:
What's causing the indirect dep here, won't we get security alerts on the repo anyway? 🤔

Reply:
Nice catch 👍  it's coming from the outdated `github.com/imdario/mergo` dependency, I will update it there.